Makefile

```diff
@@ -739,6 +746,10 @@ define fast_test_folder
 	$(foreach dir, $(wildcard $(1)/*/.), $(call fast_test_folder, $(dir)))
 endef
 
+## ####################################
```

Comment:
This should be before `define test_folder`

Reply:
Even before `build:`

Reply:
ohh you're right!